lib/cartopy/mpl/geoaxes.py

```diff
@@ -1547,6 +1547,32 @@ def scatter(self, *args, **kwargs):
         self.autoscale_view()
         return result
 
+    @_add_transform
+    def hexbin(self, x, y, *args, **kwargs):
+        """
+        Add the "transform" keyword to :func:`~matplotlib.pyplot.hexbin'.
+
+        Other Parameters
+        ----------------
+        transform
+            A :class:`~cartopy.crs.Projection`.
+        """
+        t = kwargs.get('transform', None)
```

Comment:
What about just popping this here and doing the `.transform_points()` function every single time?

Reply:
should be done now.